src/frontend/src/lib/styles/global/colors.scss

```diff
@@ -30,4 +30,5 @@
 
 	// Theme
 	--color-primary: var(--color-blue-ribbon);
+	--color-primary-rgb: 0, 102, 255;
```

Comment:
Should we rather define the tailwind color in rgb that way we don't hardcode the equivalent in css?

Reply:
Or maybe there is a way to derive the rgb?

Reply:
there is a way 

`from var(--color-blue-ribbon) r g b;`

but I have to check if it's compatible all over. Let me check and come back here